spec/models/profile_spec.rb

```diff
@@ -1016,33 +1016,20 @@
   # TODO: related: should we test against an active profile here?
   describe '#deactivate_for_fraud_review' do
     it 'sets fraud_review_pending to true and sets fraud_pending_reason' do
-      profile = create(:profile, user: user)
-
-      expect(profile.activated_at).to be_nil
-      expect(profile.active).to eq(false) # ???
-      expect(profile.deactivation_reason).to be_nil
-      expect(profile.fraud_review_pending?).to eq(false) # to change
-      expect(profile.gpo_verification_pending_at).to be_nil
-      expect(profile.in_person_verification_pending_at).to be_nil
-      expect(profile.initiating_service_provider).to be_nil
-      expect(profile.verified_at).to be_nil
+      profile = create(:profile, :in_person_verification_pending, user: user)
 
       profile.fraud_pending_reason = 'threatmetrix_review'
-      profile.deactivate_for_fraud_review
-
-      expect(profile.activated_at).to be_nil
-      expect(profile.active).to eq(false)
-      expect(profile.deactivation_reason).to be_nil
-      expect(profile.fraud_review_pending?).to eq(true) # changed
-      expect(profile.gpo_verification_pending_at).to be_nil
-      expect(profile.in_person_verification_pending_at).to be_nil
-      expect(profile.initiating_service_provider).to be_nil
-      expect(profile.verified_at).to be_nil
+      expect { profile.deactivate_for_fraud_review }.to change {
+                                                          profile.fraud_review_pending?
+                                                        }.from(false).to(true).
+        and change {
+          profile.in_person_verification_pending_at
+        }.to(nil)
```

Comment:
`rubocop -A` made an absolute abomination of this formatting. I tried to clean it up by hand, but rubocop wasn't having it.

@zachmargolis You had provided an example that was considerably less ugly. Do you have any thoughts on how to make this look more like that without rubocop rejecting it?